### ISSUE TITLE
Correctly label home page link in design system header

### ISSUE THREAD

**Issue by NicolaSaunders** [w3c/w3c-website#114]
Because the inline SVG with the W3C logo has the `aria-hidden="true"` attribute, it is not announced within the home page link, meaning that screen readers will only announce "design system" rather than "W3C design system".

The easiest way to solve this is to amend the `span` holding the text "Design System" as follows:

```
<span><span class="visuallyhidden">W3C</span> Design System</span>
```

**Comment by iadawn** [comment]
Or change the SVG to:

```<svg xmlns="http://www.w3.org/2000/svg" focusable="false" overflow="visible" viewBox="0 0 72 48" width="72px" height="48px" role="img" aria-label="W3C">... ```

**Comment by NicolaSaunders** [comment]
Pull request has been raised: https://github.com/w3c/w3c-website-templates-bundle/pull/100

**Comment by deniak** [comment]
Fixed by https://github.com/w3c/w3c-website-templates-bundle/pull/100
